### PR TITLE
feat: add pause deposits script

### PR DESCRIPTION
**Pull request by md0x**
Changes proposed in this PR:
- Add pause deposits script

### CODE REVIEW

File: scripts/svm/remoteHubPoolPauseDeposits.ts [hunk 1]
```diff
@@ -0,0 +1,211 @@
+// This script bridges remote call to pause deposits on Solana Spoke Pool. Required environment:
+// - ETHERS_PROVIDER_URL: Ethereum RPC provider URL.
```
Comment (chrismaree) [inline scripts/svm/remoteHubPoolPauseDeposits.ts:2]: NIT: we should think about cleaning these up to be more consistent with other scripts. this is a unique variable that is not named like this in other places. might be ok, for now but as we make these more mature it should standardize.

File: scripts/svm/remoteHubPoolPauseDeposits.ts [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  const seed = new BN(0);
+  const resumeRemoteTx = resolvedArgv.resumeRemoteTx;
+  const pause = resolvedArgv.pause;
+
+  // Set up Ethereum provider.
+  if (!process.env.ETHERS_PROVIDER_URL) {
+    throw new Error("Environment variable ETHERS_PROVIDER_URL is not set");
+  }
+  const ethersProvider = new ethers.providers.JsonRpcProvider(process.env.ETHERS_PROVIDER_URL);
+  if (!process.env.ETHERS_MNEMONIC) {
```
Comment (chrismaree) [inline scripts/svm/remoteHubPoolPauseDeposits.ts:42]: https://github.com/across-protocol/contracts/blob/chrismaree/add-missing-events/scripts/deployZkSyncSpokePoolProxy.ts#L16

this structure might be better

Comment (chrismaree) [inline scripts/svm/remoteHubPoolPauseDeposits.ts:42]: https://github.com/across-protocol/contracts/blob/chrismaree/add-missing-events/scripts/deployZkSyncSpokePoolProxy.ts#L16

this structure might be better

File: scripts/svm/remoteHubPoolPauseDeposits.ts [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+async function remoteHubPoolPauseDeposit(): Promise<void> {
+  const resolvedArgv = await argv;
+
+  const chainId = resolvedArgv.chainId;
+  const seed = new BN(0);
+  const resumeRemoteTx = resolvedArgv.resumeRemoteTx;
+  const pause = resolvedArgv.pause;
+
+  // Set up Ethereum provider.
+  if (!process.env.ETHERS_PROVIDER_URL) {
```
Comment (chrismaree) [inline scripts/svm/remoteHubPoolPauseDeposits.ts:38]: also this is diffrent to how the rest of the repo does this. either you should should use `CUSTOM_NODE_URL` or use `getNodeUrl` like done in other places

**Review by Reinis-FRP — APPROVED** [review]
confirm that the script is working as expected for me

**Pull request merged**